utoipa-gen/tests/schema_derive_test.rs

```diff
@@ -261,6 +261,81 @@ fn derive_struct_with_default_attr() {
     };
 }
 
+#[test]
+fn derive_struct_with_default_attr_field() {
+    struct Book;
+    let owner = api_doc! {
+        struct Owner {
+            #[schema(default = json!({ "name": "Dune" }))]
```

Comment:
I assume this example actually should be `json!([{ "name": "Dune" }])`

Reply:
@juhaku my bad. `Owner::favorite_books` was to test the bare user-defined type, not the `Vec<>`. Fixed in c27d99471bf9533629920d30c610c36154814b03

Reply:
Good